Airports/EGNV/Sector.txt

```diff
@@ -3,8 +3,11 @@ DISPLAY:NVAPP:NVAPP:NVTWR
 DISPLAY:NVTWR:NVAPP:NVTWR
 
 SECTORLINE:NVAPP
+DISPLAY:NVAPP:East:NVAPP
+DISPLAY:NTAPP:NTAPP:NVAPP
```

Comment:
Do we need both a ring and this line, or just one of them?

Reply:
I think showing the actual sector is probably more useful than just a circle, so let's go for that

Reply:
Yes, just one final thing....